pkg/trace/api/debugger.go

```diff
@@ -0,0 +1,99 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package api
+
+import (
+	"fmt"
+	stdlog "log"
+	"net/http"
+	"net/http/httputil"
+	"net/url"
+	"strings"
+	"time"
+
+	"github.com/DataDog/datadog-agent/pkg/config"
+	"github.com/DataDog/datadog-agent/pkg/trace/info"
+	"github.com/DataDog/datadog-agent/pkg/trace/logutil"
+	"github.com/DataDog/datadog-agent/pkg/trace/metrics"
+	"github.com/DataDog/datadog-agent/pkg/util/fargate"
+)
+
+const (
+	// logsIntakeURLTemplate specifies the template for obtaining the profiling URL along with the site.
+	logsIntakeURLTemplate = "https://http-intake.logs.%s/v1/input"
+	// logsIntakeDefaultURL specifies the default intake API URL.
+	logsIntakeDefaultURL = "https://http-intake.logs.datadoghq.com/v1/input"
+
+	// debuggerRequestMetricsPrefix is the prefix for the following debugger metric names.
+	debuggerRequestMetricsPrefix       = "datadog.trace_agent.debugger."
+	debuggerRequestCountMetricsName    = debuggerRequestMetricsPrefix + "proxy_request"
+	debuggerRequestDurationMetricsName = debuggerRequestMetricsPrefix + "proxy_request_duration_ms"
+	debuggerRequestErrorMetricsName    = debuggerRequestMetricsPrefix + "proxy_request_error"
```

Comment:
We don't use constants for metrics anywhere in `pkg/trace`. I know you copied these from the AppSec package. It happens there because I gave them freedom to use whatever coding style they want since it's a different team. Writing this now I realise it's not the best practice and there should be repository-wide coding conventions, but it is what it is. Can you please just inline them as strings?